app/src/main/java/com/example/vybe/MainActivity.java

```diff
@@ -40,13 +41,19 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+
         loginButton = findViewById(R.id.login_button);
         signupButton = findViewById(R.id.signup_button);
         emailField = findViewById(R.id.email_edit_text);
         passwordField = findViewById(R.id.password_edit_text);
 
+        emailField.setText("test2@vybe.ca");
+        passwordField.setText("testuser2");
+
         mAuth = FirebaseAuth.getInstance();
 
+//        mAuth.signOut();
```

Comment:
what is this?

Reply:
Once you sign in, it remembers you. So you have to manually tell it to sign out. But I'll get rid of the comment